jtelegrambotapi-core/src/main/java/com/jtelegram/api/update/UpdateType.java

```diff
@@ -79,6 +79,10 @@
             "MESSAGE",
             Update.MessageUpdate.class,
             (bot, update) -> {
+                if (update.getMessage() == null) {
```

Comment:
While there's nothing inherently wrong with this, it just doesn't make sense. The whole idea of this is that message isn't null and that's why it's called. The code literally checks if the field exists before calling its callback ([source](https://github.com/jTelegram/jTelegramBotAPI/blob/master/jtelegrambotapi-core/src/main/java/com/jtelegram/api/update/Update.java#L76-L78)). If the field 'message' didn't exist, it would be serialized as a generic Update and the callback would throw a cast exception.

The case may be that Telegram explicitly states the message is null, but that would raise the question: Why are they sending it in the first place?

It's probably worth looking into this as a bug instead of doing this, could you create an issue with information about how this happened on your bot?

Reply:
This is no longer necessary since #23 was fixed, so I'll get rid of it.